lib/location/Kconfig

```diff
@@ -0,0 +1,147 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig LOCATION
+	bool "Location"
+	depends on NRF_MODEM_LIB
+	help
+	  TODO
```

Comment:
Need to write something here.

Reply:
Removed actually after browsing though the code base and seeing that it's also empty in many cases.